modules/literatureannotation/src/main/resources/schemas/neurosciencegraph/literatureannotation/parameterannotation/v0.2.0.json

```diff
@@ -0,0 +1,61 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/literatureannotation/parameterannotation/v0.2.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/core/annotation/v0.2.0"
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "ParameterAnnotationShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:ParameterAnnotation",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/annotation/v0.2.0/shapes/AnnotationShape"
+        },
+        {
+          "property": [
+            {
+              "path": "oa:hasTarget",
+              "sh:xone": [
+                {
+                  "node": "{{base}}/schemas/neurosciencegraph/commons/annotation/v0.2.0/shapes/TextPositionTargetShape",
+                  "class": "nsg:TextPositionTarget"
+                },
+                {
+                  "node": "{{base}}/schemas/neurosciencegraph/commons/annotation/v0.2.0/shapes/FigureTargetShape",
+                  "class": "nsg:FigureTarget"
+                },
+                {
+                  "node": "{{base}}/schemas/neurosciencegraph/commons/annotation/v0.2.0/shapes/EquationTargetShape",
+                  "class": "nsg:EquationTarget"
+                },
+                {
+                  "node": "{{base}}/schemas/neurosciencegraph/commons/annotation/v0.2.0/shapes/TableTargetShape",
+                  "class": "nsg:TableTarget"
+                },
+                {
+                  "node": "{{base}}/schemas/neurosciencegraph/commons/annotation/v0.2.0/shapes/AreaTargetShape",
+                  "class": "nsg:AreaTarget"
+                }
+              ],
+              "maxCount": 1
```

Comment:
As far as I understood, the Web Annotation Data Model (https://www.w3.org/TR/annotation-model/#bodies-and-targets) specifies, that the minCount for a target is 1 while there is no minCount on the body. Should the property shape `oa:hasTarget` hence have a minCount of 1 and the minCount constraint on the `oa:hasBody` property shape below be removed?

```
The relationship between an Annotation and its Target. 
There must be 1 or more target relationships associated with an Annotation.
```


```
The relationship between an Annotation and its Body. 
There should be 1 or more body relationships associated with an Annotation but there may be 0.
```